### PR TITLE
Suggest to remove files with wp core checksum

### LINKED ISSUE #102: Cruft remover: suggest site admins to remove extra files found by wp core checksum
On many sites the command `wp core checksum` (and `wp-fix-checksums`) finds extra files. Examples:
```
Warning: File should not exist: wp-config-old.php
Warning: File should not exist: wp-config-tmp.php

Warning: File should not exist: wp-admin/XXX_YYY_2018-03-06T10:18:42+00:00.sql
Warning: File should not exist: wp-admin/XXX_YYY_2018-03-06T10:17:58+00:00.sql
Warning: File should not exist: wp-admin/XXX_YYY_2018-03-06T09:41:42+00:00.sql
Warning: File should not exist: wp-admin/XXX_YYY_2018-02-10T09:47:39+00:00.sql
Warning: File should not exist: wp-admin/XXX_YYY_2018-02-13T09:35:48+00:00.sql

Warning: File should not exist: wp-includes/wp-feed.php
Warning: File should not exist: wp-includes/wp-tmp.php

Warning: File should not exist: wp-admin/XXX_YYY_2018-02-28T07:55:48+00:00.sql
Warning: File should not exist: wp-admin/XXX_YYY_2018-02-19T12:59:37+00:00.sql
Warning: File should not exist: wp-admin/XXX_YYY_2018-02-28T07:53:31+00:00.sql
Warning: File should not exist: wp-admin/XXX_YYY_2018-02-28T07:54:32+00:00.sql
Warning: File should not exist: wp-admin/XXX_YYY_2018-02-19T12:58:54+00:00.sql
Warning: File should not exist: wp-admin/XXX_YYY_2018-02-19T13:00:25+00:00.sql
Warning: File should not exist: wp-admin/XXX_YYY_2018-04-16T11:17:44+00:00.sql

Warning: File should not exist: wp-config-development-sample.php
Warning: File should not exist: wp-config-tmp.php
Warning: File should not exist: wp-config-local-sample.php
```

(XXX_YYY is instead of actual site/customer names)

The cruft remover should suggest removing all of these files as well. If there is a plugin/theme/developer who likes to add their own files into WordPress core directories, they should stop doing that.

### PR DESCRIPTION
**Pull request by mikkoaf**
Closes #102

### CODE REVIEW

**Comment by ottok** [comment]
It seems the git author name is not configured correctly, see https://github.com/Seravo/seravo-plugin/commit/6473426838742a43a6c56d34ed90d77e118a14f7.patch and fix instructions at https://github.com/Seravo/seravo-plugin/pull/109#issuecomment-392007726

**Comment by ottok** [comment]
Other than the thing above this looks good to me. In the future there could be n UI indication on _why_ these files are cruft and should be removed (=incorrectly placed in wordpress/ while not being a file that is part of WP core) but for now this PR is fine and a step forward.

**Comment by l3ku** [comment]
I will review this after https://github.com/Seravo/seravo-plugin/pull/105 has been merged to master and this branch rebased against the new modifications in master.

File: lib/cruftfiles-ajax.php [hunk 1]
```diff
@@ -10,6 +10,21 @@ function find_cruft_dir( $name ) {
...
 
+function find_cruft_core() {
+  $output = array();
+  $handle = popen('wp core verify-checksums 2>&1', 'r');
+  $temp = stream_get_contents($handle);
+  pclose($handle);
+  // Lines beginning with: "Warning: File should not exist: "
+  $temp = explode("\n", $temp);
+  foreach ( $temp as $line ) {
+    if(strpos($line,"Warning: File should not exist: ") !== false) {
```
Comment (l3ku) [inline lib/cruftfiles-ajax.php]: Please add whitespace in if clauses, e.g.
```
if ( $page === 'tools_page_database_page' ) {
```

File: lib/cruftfiles-ajax.php [hunk 2]
```diff
@@ -10,6 +10,21 @@ function find_cruft_dir( $name ) {
...
+      $line = ["/data/wordpress/htdocs/wordpress/" . substr($line, 32)];
```
Comment (l3ku) [inline lib/cruftfiles-ajax.php]: Is it necessary to introduce a single line inside an array?

**Pull request merged**